### PR TITLE
Separate secrets from main config #8

### PR DESCRIPTION
**Pull request by DaniilGo**
resolves: https://github.com/CSR-LC/issues/issues/8

### CODE REVIEW

File: .github/workflows/blank.yml [hunk 1]
```diff
@@ -95,6 +95,12 @@ jobs:
           script: |
             mkdir -p /var/www/csr/stage/
             echo '${{ secrets.DEPLOY_CONFIG }}' > /var/www/csr/stage/config.json
+            sudo rm -rf /etc/systemd/system/stage.csr.service.d
+            sudo EDITOR='tee' systemctl edit stage.csr.service <<< '[Service]
```
Comment (antelman107) [inline .github/workflows/blank.yml:99]: Is it possible to just overwrite this file like 
`echo 'filecontent' > service_file`?
`EDITOR='tee' systemctl edit` looks complicated.
I know it's mentioned in stackoverflow link I sent earlier, but the main idea from that link was `Environment=`, not the exact process of how the file was edited.

**Comment by antelman107** [comment]
Since you are changing a CI/CD pipeline, can you attach a link to the pipeline is successfully working?

File: .github/workflows/blank.yml (hunk 1 above)
Comment (DaniilGo) [reply .github/workflows/blank.yml:99]: In the current case we will create dir /etc/systemd/system/stage.csr.service.d and file owerride.conf, which will contains our env vars. By the way, I tested different approaches - this one works fine

Comment (DaniilGo) [reply .github/workflows/blank.yml]: You are right @antelman107! I will try to make the flow more simple

**Comment by DaniilGo** [comment]
> Since you are changing a CI/CD pipeline, can you attach a link to the pipeline is successfully working? (it's a non-master branch, so you need to modify the conditions of the pipeline)

Here it is: https://github.com/CSR-LC/csr-be/actions/runs/11059612897/job/30728582557

**Review by WolfusFlow — APPROVED** [review]

File: .github/workflows/blank.yml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: build-deploy
 on:
   workflow_dispatch:
   push:
-    branches: ["master"]
+    branches: ["feat/master"]
```
Comment (antelman107) [inline .github/workflows/blank.yml:5]:
```suggestion
    branches: ["master"]
```
right?

File: .github/workflows/blank.yml [hunk 3]
```diff
@@ -95,6 +95,17 @@ jobs:
           script: |
             mkdir -p /var/www/csr/stage/
             echo '${{ secrets.DEPLOY_CONFIG }}' > /var/www/csr/stage/config.json
+            sudo rm -rf /etc/systemd/system/stage.csr.service.d
```
Comment (antelman107) [inline .github/workflows/blank.yml:98]: I think `EnvironmentFile` directive might be simplier to work with compared to folder management.
https://www.flatcar.org/docs/latest/setup/systemd/environment-variables/
Could you try this approach and see how it works?
Please create environment file in the same directory with `stage.csr.service` - do not create unnecessary folders as possible.

**Review by antelman107 — CHANGES_REQUESTED** [review]
The solution is working - good, but env file management could be optimized - https://github.com/CSR-LC/csr-be/pull/114/files#r1778498849

File: .github/workflows/blank.yml (hunk 2 above)
Comment (DaniilGo) [reply .github/workflows/blank.yml]: Thanks, fixed

File: .github/workflows/blank.yml (hunk 3 above)
Comment (DaniilGo) [reply .github/workflows/blank.yml]: Thanks for your advice! Updated this flow. Here are successful run: https://github.com/CSR-LC/csr-be/actions/runs/11105969158

**Comment by DaniilGo** [comment]
Now looks good (IMHO) and works fine, thanks!

**Review by antelman107 — APPROVED** [review]
👍

**Pull request merged**